CHANGELOG.txt

```diff
@@ -330,6 +330,9 @@
                 - "creme_core/bricks/generic/hat-bar.html"
         # Javascript :
             - Removed ChainedSelect.clone() method : not used and does not work anyway.
+            - Refactor creme.notification.NotificationBox & add unit tests
+                - Toggle fetch 'job' when disabling a tab or window.
+                - Elapsed time from message creation is now updated each minute.
```

Comment:
- I did not understand the "toggle" improvement.
- It's seems the second one (at least) is an interesting user-side improvement .

Reply:
the fetching job is disabled when the browser tab is not active